README.md

```diff
@@ -2176,6 +2177,32 @@ you are using apply and auto correct.
 - [CMSiteConfiguration_Cas](Source\Examples\Resources\CMSiteConfiguration\CMSiteConfiguration_Cas.ps1)
 - [CMSiteConfiguration_Primary](Source\Examples\Resources\CMSiteConfiguration\CMSiteConfiguration_Primary.ps1)
 
+### CMHierarchySetting
```

Comment:
Please add the new resource to supported resources under the reverse dsc section

Reply:
Done